src/components/Button/ButtonLabel.tsx

```diff
@@ -5,6 +5,9 @@ import typography from "../../styles/utils/typography";
 import { Span } from "../Typography";
 
 const ButtonLabel = styled(Span)`
+  display: inline-block;
```

Comment:
Again a bit concerned seeing these sorts of unexplained style changes at such a low level. Have you checked for unintended effects on other buttons ?

Reply:
Paired with @mantagen on this as the buttons weren't behaving as expected when text was long the button / container wouldn't shrink with the resizing of the page, which was causing buttons with long text to overlap with other components. This is to ensure the ellipsis appears as button shrinks and text overflows